YapDatabase/YapDatabase.m

```diff
@@ -231,7 +234,10 @@ + (YapDatabaseDeserializer)timestampDeserializer
 		}
 		else
 		{
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
 			return [NSKeyedUnarchiver unarchiveObjectWithData:data];
+#pragma clang diagnostic pop
```

Comment:
I'm guessing this can't be updated for the same reasons as above?

Reply:
Yep.